src/util/psbt/error.rs

```diff
@@ -32,7 +32,7 @@ pub enum PsbtHash {
     Hash256,
 }
 /// Ways that a Partially Signed Transaction might fail.
-#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Hash, Debug)]
```

Comment:
I'm going to leave this as is since this PR only _removes_ `Ord`. The correct fix is to audit all the error types we have and use a fixed, minimal, set of derives, right?

Reply:
Yes.